1-Draft/RFCXXXX-ScriptBlockLogging.md

```diff
@@ -0,0 +1,143 @@
+---
+RFC: RFCXXXX
+Author: Travis Plunk
+Status: Draft
+SupercededBy:
+Version: 1.0
+Area: Engine
+Comments Due: 08/15/2019
+Plan to implement: Yes
+---
+
+# Title
+
+ScriptBlock Logging
+
+## Description
+
+This is an addition to the existing ScriptBlock creation logging.
+The goal is to make it easier to configure to collect the logs centrally.
+This feature will be experimental.
+
+## Motivation
+
+    As a administrator of machines which run PowerShell (machine admins),
+    I can easily log stript blocks which PowerShell executes to systems such as Azure Log Analytics or Splunk (log systems),
+    so that I can analyze PowerShell activity on all the machines I run.
+
+## User Experience
+
+This example assumes you have registered the key to communicate with Azure Log Analytics with PowerShell Secrets with the name `myKeyName`.
+
+```powershell
+Register-PSScriptBlockLogger -Provdier AzureLogAnalytics -KeyName myKeyName -Properties @{
+    WorkspaceId = ‘myworkspaceid’
+}
+```
+
+```output
```

Comment:
-Get and -Unregister cmdlets could be helpful

What is the error message or warning if you try to register a second, a -Force parameter?